packages/graphql/package.json

```diff
@@ -1,10 +1,26 @@
 {
   "name": "@typespec/graphql",
   "version": "0.1.0",
+  "author": "Microsoft Corporation",
```

Comment:
Is this true?

Reply:
As [I mentioned](https://github.com/pinterest/typespec/pull/1#issuecomment-2471795761), these `package.json` updates have been proposed upstream on their own. So I am hoping to get MSFT's input on what should and shouldn't be in `package.json` as I think it's ultimately up to them.

Until that, I am mostly just copying things over from other packages (primarily `openapi3`).